File: message-storage-service/src/main/java/com/yazikochesalna/messagestorageservice/MessageStorageServiceApplication.java
package com.yazikochesalna.messagestorageservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication (scanBasePackages = {
        "com.yazikochesalna.messagestorageservice",
        "com.yazikochesalna.common"
})
public class MessageStorageServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(MessageStorageServiceApplication.class, args);
    }

}

File: message-storage-service/src/main/kotlin/com/yazikochesalna/messagestorageservice/MessageStorageServiceApplication.kt
package com.yazikochesalna.messagestorageservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(exclude = [DataSourceAutoConfiguration::class], scanBasePackages =
[
    "com.yazikochesalna.messagestorageservice",
    "com.yazikochesalna.common",
])
open class MessageStorageServiceApplication

fun main(args: Array<String>) {
    runApplication<MessageStorageServiceApplication>(*args)
}